src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/PrisonerActivitiesCountRepository.kt

```diff
@@ -16,8 +17,9 @@ interface PrisonerActivitiesCountRepository : CrudRepository<PrisonerActivitiesC
   @Query(
     """
         SELECT 
-          COUNT(*) as total,
```

Comment:
When this was run in production it was exceedingly slow, even though in pre-prod is was quite fast.
I've therefore switched to the existing method of retrieving the bookings and then filtering in the java / kotlin instead in the hope that it helps